js/razorpay/razorpay-utils.js

```diff
@@ -5,47 +5,47 @@ function RazorpayUtils(options) {
         return this.options.merchant_name;
     },
 
-    this.placeOrder = function(onSuccess, onDismiss, formId, paymentIdField)
-    {
-        if (!formId) {
-            formId = "razorpay";
-        }
-        if (!paymentIdField) {
-            paymentIdField = "razorpay_payment_id";
-        }
+        this.placeOrder = function(onSuccess, onDismiss, formId, paymentIdField) {
```

Comment:
@mayankamencherla why is this change needed?

Reply:
@ccrims0n JS convention to have opening brace on the same line.

Reply:
@mayankamencherla please move this to a separate pr.